### PR TITLE
Update gzip sample with comments about GzipHander default behavior.

### PR DESCRIPTION
**Pull request by souenzzo**

### CODE REVIEW

**Comment by ohpauleez** [comment]
Thanks!  It's a useful suggestion.  I just need someone on the Pedestal team to verify the suggestions work and we can get this merged.

**Comment by ddeaguiar** [comment]
@ohpauleez, I had discussed this with @souenzzo in the #pedestal slack room. I'll take a closer look and merge if ok soonish.

File: samples/servlet-filters-gzip/src/gzip/service.clj [hunk 1]
```diff
@@ -61,6 +61,11 @@
               ::http/container-options {:context-configurator (fn [c]
                                                                 (let [gzip-handler (GzipHandler.)]
                                                                   (.setGzipHandler c gzip-handler)
+                                                                  ;; by default, "GzipHandler" only operates on "GET" 's.
+                                                                  ;; It is possible to add other methods via:
+                                                                  ;; (.addIncludedMethods gzip-handler (into-array ["GET" "POST"]))
```
Comment (ddeaguiar) [inline samples/servlet-filters-gzip/src/gzip/service.clj]: Can we remove `"GET"`? Including it here is unnecessary although it does no harm to do so.

Comment (ohpauleez) [reply samples/servlet-filters-gzip/src/gzip/service.clj]: Unnecessary, but it's nice to see it explicitly listed.

Comment (ddeaguiar) [reply samples/servlet-filters-gzip/src/gzip/service.clj]: Ok.

**Pull request merged**